src/main/java/appeng/core/sync/packets/PacketValueConfig.java

```diff
@@ -158,6 +158,10 @@ else if( this.Name.equals( "PatternTerminal.Clear" ) )
 			{
 				cpt.clear();
 			}
+			else if( this.Name.equals( "PatternTerminal.Substitute" ) )
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "PatternTerminal.Substitute" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1132)